consensus/vbft/block_pool.go

```diff
@@ -477,6 +477,7 @@ func (pool *BlockPool) newBlockCommitment(msg *blockCommitMsg) error {
 		if crossChainMsgSig, present := msg.CrossChainMsgEndorserSig[endorser]; present {
```

Comment:
check if CrossChainMsgEndorserSig is nil?

Reply:
msg_builder // 383 had made CrossChainMsgEndorserSig